pkg/build/message.go

```diff
@@ -0,0 +1,23 @@
+package build
+
+import (
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+const (
+	messageScriptName = "message.sh"
+)
+
+func ConfigureMessage(buildConfig *config.BuildConfig) error {
+	// There's currently intentionally no way to disable this in the image config,
```

Comment:
Not sure what we refer to here. Could you please explain?

Reply:
Probably overkill. There's no flag in the image config file that says "Don't drop a 'configured by'" message. It wasn't an omission to check for that flag, we just drop it in every case. But I'm overthinking things with the comment and removed it.